Src/BootCamp.Chapter/Logger/FileLogger.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.IO;
+
+namespace BootCamp.Chapter.Logger
+{
+    class FileLogger : ILogger
+    {
+        string _logPath;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_logPath' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5147695)